extra/release_tool.py

```diff
@@ -256,8 +266,8 @@ class Component:
                 "release_component": False,
             },
             "mender-useradm": {
-                "git": ["useradm"],
-                "docker_image": ["useradm"],
+                "git": ["useradm", "useradm-enterprise"],
+                "docker_image": ["useradm", "useradm-enterprise"],
                 "release_component": True,
```

Comment:
why here under mender-useradm key you put useradm-enterprise?

Reply:
I have to defer to @kacf here.
this was just a semi-intelligent copy-paste based on other components.

Reply:
just as mine yesterday :) https://github.com/mendersoftware/integration/pull/640

Reply:
It's because the `useradm-enterprise` image is a *replacement* for the `mender-useradm` container, not an extra container.